L1Trigger/L1TMuonCPPF/python/emulatorCppfDigis_cfi.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+emulatorCppfDigis = cms.EDProducer("L1TMuonCPPFDigiProducer",
+                                   
+                                   ## Input collection
+                                   recHitLabel = cms.InputTag("rpcRecHits"),
+				   MaxClusterSize = cms.int32(3),
+                                   #  cppfSource = cms.string('Geo'), #'File' for Look up table and 'Geo' for CMSSW Geometry 
+                                   cppfSource = cms.string('File'), #'File' for Look up table and 'Geo' for CMSSW Geometry 
+                                   
+                                   cppfvecfile = cms.FileInPath('L1Trigger/L1TMuonCPPF/data/angleScale_RPC_CPPFmerged.txt')                     
```

Comment:
CPPF data files will be available as externals with path `L1Trigger/L1TMuon/data/cppf/`,
effective the moment @smuzaffar gives us the externals PR#.
(https://github.com/cms-sw/cmsdist/issues/3854)

@maseguracern 
Please already adjust for it in the lines below.

Reply:
Done